client/src/components/Attachment/Attachment.css

```diff
@@ -103,10 +103,6 @@
   border-radius: 5px;
   transition: transform .5s ease;
 }
-.img-hover-zoom:hover .attachment-image {
```

Comment:
You also no longer need the `transition` from the `attachment-image` class directly above.

Reply:
Thanks, done.

Reply:
Did you push that change? The line is still there in the PR…

Reply:
Sorry... 😬